source/particles/particle_handler.cc

```diff
@@ -1745,6 +1751,21 @@ namespace Particles
           break;
       }
   }
+
```

Comment:
This needs two more empty lines to follow our coding style (3 empty lines between function definitions, 1 empty line in headers).